### ISSUE TITLE
`"hidden": true` doesn't work in areas.json

### ISSUE THREAD

**Issue by kamieniarz** [WatWowMap/ReactMap#444]
Tested on latest develop. Other properties work just fine, only `hidden` doesn't work - geofences are always displayed, doesn't matter if `hidden` is `false` or `true`

**Comment by lenisko** [comment]
Apparently, `hidden` is unhandled in the code. First place to change is a list of areas `src/components/layout/drawer/Areas.jsx` where filter would do a thing. The rest of the places are map renders.
```jsx
.filter(area => (area.properties.hidden == undefined || area.properties.hidden === false))
```

**Comment by TurtIeSocks** [comment]
Yeah this functionality doesn't actually exist anywhere. Just a lazy Copy/Paste bug. Oops. Will remove from the areas example JSON.

**Comment by lenisko** [comment]
Nah, leave it. Worth adding as I have area covering whole country which I would love to hide as well :p

**Comment by TurtIeSocks** [comment]
if you want it then I will PR it tonight 💋

**Comment by lenisko** [comment]
Aww... Thanks :3

**Comment by TurtIeSocks** [comment]
Resolved in #445
